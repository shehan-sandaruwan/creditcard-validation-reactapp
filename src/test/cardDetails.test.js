import React from "react";
import { shallow } from "enzyme";
import CardDetails from "../Component/cardDetails";

import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("CardDetails", () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<CardDetails debug />);

    expect(component).toMatchSnapshot();
  });
});
