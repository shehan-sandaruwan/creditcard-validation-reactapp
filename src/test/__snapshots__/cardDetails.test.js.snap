// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardDetails should render correctly in "debug" mode 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
